src/instructlab/cli/process/prune.py

```diff
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Third Party
+import click
+
+# First Party
+from instructlab import clickext
+from instructlab.defaults import ILAB_PROCESS_STATUS
+from instructlab.process.process import (
+    display_processes,
+    filter_process_with_conditions,
+    remove_process,
+)
+
+
+@click.command(name="prune")
+@click.option(
+    "--older",
+    type=click.INT,
+    help="The day older than the specified number of days. Use 0 to remove all.",
```

Comment:
`The day older than the specified number of days` is not very clear. Consider `Match processes that are older than the specified number of days`?